test/integration/util/util.go

```diff
@@ -43,6 +45,17 @@ func (m *MinikubeRunner) RunCommand(command string, checkError bool) string {
 	return string(stdout)
 }
 
+func (m *MinikubeRunner) SetEnvFromEnvCmdOutput(dockerEnvVars string) {
+	//TODO: aprindle: output errors
+	lines := strings.Split(dockerEnvVars, "\n") //5 lines
+	instructionLines := 2
```

Comment:
What about instead of hardcoding this you just set all lines that match the format string in fmt.Sscanf? I think you can check the error and return codes to determine if it was valid.